pennylane/ops/qubit/special_unitary.py

```diff
@@ -123,6 +124,15 @@ def pauli_words(num_wires):
     return ["".join(letters) for letters in list(product(_pauli_letters, repeat=num_wires))[1:]]
 
 
+def _detach(array, interface):
+    """Detach an array from its trace and return just its numerical values."""
+    if interface == "jax":
+        import jax
+
+        return jax.lax.stop_gradient(array)
+    return qml.math.convert_like(qml.math.to_numpy(array), array)
```

Comment:
Resolving, because what comes after using `_detach` anyways will be much slower than `_detach` itself:
For every array entry we create and queue a new operation.